### ISSUE TITLE
Can you help me with terms

### ISSUE THREAD

**Issue by Fluffy728** [tunderwood89/FireShop#6]
It broke. And, I Forget how to fix it.

**Comment by tunderwood89** [comment]
> It broke. And, I Forget how to fix it.

Don't worry I think I have a backup!

**Comment by Fluffy728** [comment]
It is still broken

**Comment by tunderwood89** [comment]
> It is still broken

Yeah I don't have a backup. I was gonna start using a site builder anyway.
